### DISCUSSION TITLE
Wireguard Setup.

### DISCUSSION THREAD

**Discussion by chop249** [Ysurac/openmptcprouter#3613] [General]
I had a VPS and Router working together but then went to update and everything went sideways. I have a new VPS and 2 routers set up now. On my old router though I had the WireGuard settings taken from the VPS and placed those in the router somewhere. I cannot find where I found that information a couple years ago when I got it working. The real reason I want to do it is because I have PiHoles set up on my LAN and when I set up the WG last time ad blocking was transferred to the OMR as well. It is not being transferred currently and I know I can run the ad block DNS servers through OMR but I like the way my PiHoles handle blocking better than those. It really doesn't NEED to happen through WG if someone can tell me how to force DNS through the LAN where the VPS is. I can access all local services and machines as though I was on the LAN, just not getting routed through the DNS for whatever reason.

**Comment by Ysurac** [comment]
I'm not sure to know what you are doing with WG.
You can set custom DNS in Network->DHCP & DNS ( https://github.com/Ysurac/openmptcprouter/wiki/Custom-DNS )
